src/polymer-project.ts

```diff
@@ -181,18 +173,18 @@ export class PolymerProject {
     });
   }
 
-  // TODO(justinfagnani): add options, pass to vfs.src()
   dependencies(): NodeJS.ReadableStream {
-    let deps = this.dependencyGlobs;
-    return vfs.src(deps, {
-      allowEmpty: true,
-      cwdbase: true,
-      nodir: true,
-    });
+    return this.sources()
```

Comment:
I think this could be something like `return this._analyzer.dependencyPaths()`, where dependencyPaths() is fed files from `StreamResolver.accept()`.